Fix cilium XDP demo

It looks like after moving to nuget package, the XDP demo does not work on the final test setup. Compilation of Cilium XDP program fails due to
1. missing header files
2. Some redefinitions.

**Status**: Reproduced.

Section: **Compiling / Loading eBPF Program**
```
PS C:\ebpf-for-windows-demo\Debug> .\cilium_based_agent.exe --bpf-lb-mode=snat --device=ethernet
Initializing daemon with mode = snat
Using interface:
  name         = ethernet
  mtu          = 1500
  ifindex      = 8
  IPv4 address = 172.29.163.18

Cleaning up previous configuration ...
Compiling XDP eBPF program for SNAT ...
In file included from bpf_xdp.c:7:
In file included from ..\include\bpf_helpers.h:13:
..\include/ebpf_structs.h:14:10: fatal error: 'stdbool.h' file not found
#include <stdbool.h>
         ^~~~~~~~~~~
1 error generated.
Failed to compile the XDP program with error 0
Verifying the program ...
bpf_object__open failed with error 0
Initialization complete.

>$
```

With the above fix of adding #define NO_CRT, there are additional errors....

```
C:\ebpf-for-windows-demo\Debug> .\cilium_based_agent.exe --bpf-lb-mode=snat --device=ethernet
Initializing daemon with mode = snat
Using interface:
  name         = ethernet
  mtu          = 1500
  ifindex      = 8
  IPv4 address = 172.29.163.18

Cleaning up previous configuration ...
Compiling XDP eBPF program for SNAT ...
In file included from bpf_xdp.c:7:
In file included from ..\include\bpf_helpers.h:14:
..\include/ebpf_structs.h:17:23: error: redefinition of typedef 'uint8_t' is a C11 feature
      [-Werror,-Wtypedef-redefinition]
typedef unsigned char uint8_t;
                      ^
..\include\stdint.h:8:23: note: previous definition is here
typedef unsigned char uint8_t;
                      ^
In file included from bpf_xdp.c:7:
In file included from ..\include\bpf_helpers.h:14:
..\include/ebpf_structs.h:18:22: error: redefinition of typedef 'uint32_t' is a C11 feature
      [-Werror,-Wtypedef-redefinition]
typedef unsigned int uint32_t;
                     ^
..\include\stdint.h:9:22: note: previous definition is here
typedef unsigned int uint32_t;
                     ^
In file included from bpf_xdp.c:7:
In file included from ..\include\bpf_helpers.h:14:
..\include/ebpf_structs.h:19:28: error: redefinition of typedef 'uint64_t' is a C11 feature
      [-Werror,-Wtypedef-redefinition]
typedef unsigned long long uint64_t;
                           ^
..\include\stdint.h:7:28: note: previous definition is here
typedef unsigned long long uint64_t;
                           ^
3 errors generated.
Failed to compile the XDP program with error 183
Failed to compile XDP program for SNAT mode.
Failed to compile and load XDP program. error=183
PS C:\ebpf-for-windows-demo\Debug>
```

--- a/cilium/load_balancer/bpf/bpf_xdp/bpf_xdp.vcxproj
+++ b/cilium/load_balancer/bpf/bpf_xdp/bpf_xdp.vcxproj
@@ -138,6 +138,7 @@
 xcopy $(EbpfIncludePath)\* $(SolutionDir)$(Platform)\$(Configuration)\include\* /E/Y
 xcopy $(SolutionDir)cilium\load_balancer\windows_types.h $(SolutionDir)$(Platform)\$(Configuration)\include\* /E/Y
 xcopy $(SolutionDir)cilium\load_balancer\stdint.h $(SolutionDir)$(Platform)\$(Configuration)\include\* /E/Y
+xcopy $(SolutionDir)cilium\load_balancer\stdbool.h $(SolutionDir)$(Platform)\$(Configuration)\include\* /E/Y
 xcopy $(SolutionDir)cilium\load_balancer\guiddef.h $(SolutionDir)$(Platform)\$(Configuration)\include\* /E/Y
 xcopy $(SolutionDir)cilium\load_balancer\bpf\node_config_static.h $(SolutionDir)$(Platform)\$(Configuration)\bpf\ /Y
 xcopy $(SolutionDir)cilium\load_balancer\bpf\netdev_config_snat.h $(SolutionDir)$(Platform)\$(Configuration)\bpf\ /Y
@@ -182,6 +183,7 @@ xcopy $(SolutionDir)cilium\load_balancer\bpf\netdev_config_dsr.h $(SolutionDir)$
 xcopy $(EbpfIncludePath)\* $(SolutionDir)$(Platform)\$(Configuration)\include\* /E/Y
 xcopy $(SolutionDir)cilium\load_balancer\windows_types.h $(SolutionDir)$(Platform)\$(Configuration)\include\* /E/Y
 xcopy $(SolutionDir)cilium\load_balancer\stdint.h $(SolutionDir)$(Platform)\$(Configuration)\include\* /E/Y
+xcopy $(SolutionDir)cilium\load_balancer\stdbool.h $(SolutionDir)$(Platform)\$(Configuration)\include\* /E/Y
 xcopy $(SolutionDir)cilium\load_balancer\guiddef.h $(SolutionDir)$(Platform)\$(Configuration)\include\* /E/Y
 xcopy $(SolutionDir)cilium\load_balancer\bpf\node_config_static.h $(SolutionDir)$(Platform)\$(Configuration)\bpf\ /Y
 xcopy $(SolutionDir)cilium\load_balancer\bpf\netdev_config_snat.h $(SolutionDir)$(Platform)\$(Configuration)\bpf\ /Y

